jsonschema/tests/test_jsonschema_test_suite.py

```diff
@@ -398,3 +457,65 @@ def leap_second(test):
         )(test)
     ),
 )
+
+
+TestDraft202012 = DRAFT202012.to_unittest_testcase(
+    DRAFT202012.tests(),
+    DRAFT202012.optional_tests_of(name="bignum"),
+    DRAFT202012.optional_tests_of(name="ecmascript-regex"),
+    DRAFT202012.optional_tests_of(name="float-overflow"),
+    DRAFT202012.optional_tests_of(name="non-bmp-regex"),
+    DRAFT202012.optional_tests_of(name="refOfUnknownKeyword"),
+    Validator=Draft202012Validator,
+    skip=lambda test: (
+        narrow_unicode_build(test)
+        or ecmascript_regex_validation(test)
+        or skip(
+            message="ToDo: Extend validation",
```

Comment:
I'm ok if we are missing support for something here, but can you change the message or add a ticket to indicate what we are missing and what'd be required to get it?

Reply:
There are two additional cases that fail that where introduced with the latest update of the tests. I'm kind of having a hard time understanding how the dynamic scopes are suposed to work, maybe they need to be split when loading `anschors` and `dynamicAnchors`. I'm still lokking into this, for now I fixed the message.